docs/ref/request-response.txt

```diff
@@ -969,6 +969,29 @@ types of HTTP responses. Like ``HttpResponse``, these subclasses live in
     :class:`~django.template.response.SimpleTemplateResponse`, and the
     ``render`` method must itself return a valid response object.
 
+
+Custom Response classes
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If you find yourself needing a response class that Django doesn't provide,
+you can implement it with the help of :py:class:`http.HTTPStatus`::
+
+    from http import HTTPStatus
+    from django.http import HttpResponse
+
+    class HttpResponseNoContent(HttpResponse):
+        status_code = HTTPStatus.NO_CONTENT
+
+You can also use a simpler version for the case when you only need a one-off
+solution::
+
+    from http import HTTPStatus
+    from django.http import HttpResponse
+
+    def my_view(request):
+        # ...
+        return HttpResponse(status=HTTPStatus.NO_CONTENT)
```

Comment:
Maybe this second example would be better as a mention in the constructor docs further up?

```
``status`` is the :rfc:`HTTP status code <7231#section-6>` for the response. You can use Python's :py:class:`http.HTTPStatus` for meaningful aliases, such as ``http.HTTPStatus.NO_CONTENT``.
```

Reply:
Good idea, I've added a commit to move it there.
Thanks.